jetty-server/src/main/java/org/eclipse/jetty/server/handler/StatisticsHandler.java

```diff
@@ -117,6 +128,25 @@ public void onComplete(AsyncEvent event) throws IOException
             }   
         }
     };
+    
+    private final HttpChannel.Listener _shutdownListener = new HttpChannel.Listener()
+    {
+        @Override 
+        public void onResponseBegin(Request request)
+        {
+            if (_shutdown.isShutdown())
```

Comment:
Do we need this `if` statement if we only add `Connection: close` when the connector is shutting down?